tokenizer/entities.test

```diff
@@ -2,12 +2,24 @@
 
 {"description": "Undefined named entity in attribute value ending in semicolon and whose name starts with a known entity name.",
 "input":"<h a='&noti;'>",
-"output": [["StartTag", "h", {"a": "&noti;"}]]},
+"output": ["ParseError", ["StartTag", "h", {"a": "&noti;"}]]},
```

Comment:
Hmmmmm. That should probably switch to the ambiguous ampersand state. https://html.spec.whatwg.org/#syntax-attribute-value says 

> Attribute values are a mixture of text and character references, except with the additional restriction that the text cannot contain an ambiguous ampersand.